packages/lib/src/vis/shared/Annotation.tsx

```diff
@@ -1,28 +1,39 @@
+import { useThree } from '@react-three/fiber';
 import type { CSSProperties, HTMLAttributes } from 'react';
+import { Vector2 } from 'three';
 
+import { useWorldToHtml } from '../hooks';
 import { useAxisSystemContext } from './AxisSystemContext';
 import Overlay from './Overlay';
 
 interface Props extends HTMLAttributes<HTMLDivElement> {
   x: number;
   y: number;
-  z?: number;
   scaleOnZoom?: boolean;
   style?: CSSProperties;
```

Comment:
Don't think the `style` prop is needed since it is already included by the base interface. You may want to spread it on the inner `div` styles, though (as per my other comment).